test/upgrade/test_main.go

```diff
@@ -24,14 +24,13 @@ import (
 	"knative.dev/eventing-kafka/test"
 	eventingTest "knative.dev/eventing/test"
 	testlib "knative.dev/eventing/test/lib"
-	"knative.dev/eventing/test/lib/resources"
 	"knative.dev/pkg/system"
 )
 
 var (
 	channelTestRunner  testlib.ComponentsTestRunner
 	defaultChannelType = metav1.TypeMeta{
-		APIVersion: resources.MessagingAPIVersion,
+		APIVersion: "messaging.knative.dev/v1beta1",
```

Comment:
@cardil I am doing this here , now....

Reply:
Right, but should be better to have that in some variable?